voice/bxml/callbacks/answer.md

```diff
@@ -21,7 +21,7 @@ Content-Type: application/xml; charset=utf-8
 | accountId     | The user account associated with the call. |
 | applicationId | The id of the application associated with the call. |
 | to            | The phone number that received the call, in E.164 format (e.g. +15555555555). |
-| from          | The phone number that made the call, in E.164 format (e.g. +15555555555). |
+| from          | The provided identifier of the caller: can be a phone number in E.164 format (e.g. +15555555555) or a string such as `Private` or `Restricted`. |
```

Comment:
Worth enumerating all possible non-E164 words like you did in other pages?
```suggestion
| from          | The provided identifier of the caller. Possible values: a phone number in E.164 format (e.g. +15555555555), `Private`, `Restricted`, .... |
```

Reply:
you're right. I wasn't think about this like "oh, Bandwidth created this call so the `from` field is necessarily limited."